### ISSUE TITLE
Type IFormGroup<FormModel> is not assignable to type FormGroup 

### ISSUE THREAD

**Issue by Antoniossss** [rxweb/rxweb#426]
# Describe the bug
When using IFormGroup and binding to `[formGroup]` template validation shows following error
Type IFormGroup<FormModel> is not assignable to type FormGroup 

# Expected Behavior
No error at all (works in rutnime obviously)

# To Reproduce
component:
`selectorForm: IFormGroup<YourType>;`

template
`<div [formGroup]="selectorForm" >`

![image](https://user-images.githubusercontent.com/7877627/96976848-9c457f80-151c-11eb-8cef-9e0d3048e91a.png)

# Package Version
@rxweb/types@1.0.6

**Comment by ajayojha** [comment]
@Antoniossss which angular version you are using?

**Comment by Antoniossss** [comment]
"@angular/animations": "~10.1.5",
    "@angular/cdk": "^10.2.4",
    "@angular/common": "~10.1.5",
    "@angular/compiler": "~10.1.5",
    "@angular/core": "~10.1.5",
    "@angular/forms": "~10.1.5",
    "@angular/material": "^10.2.4",
    "@angular/platform-browser": "~10.1.5",
    "@angular/platform-browser-dynamic": "~10.1.5",
    "@angular/router": "~10.1.5",
    "@graphy/content.nt.read": "^4.3.3",
    "@graphy/content.ttl.read": "^4.3.3",
    "@graphy/memory.dataset.fast": "^4.3.3",
    "@rxweb/types": "^1.0.6",

**Comment by ajayojha** [comment]
@Antoniossss I have tried in my local machine and it's working fine, I have created a [git repo](https://github.com/ajayojha/rxweb-type-form-group) of the respective working example for your reference, please check the same and let me know.

**Comment by Antoniossss** [comment]
Iv checked out your example and it is exactly the same.

**Comment by Antoniossss** [comment]
I have just noticed that making generic token as `any` makes warning go away.

**Comment by ristomatti** [comment]
Same issue with Angular 9.1.1. I wonder if this would cause an error and not a warning if strict mode was enabled? Our project is way too far to enable it so I can't easily test this.

**Comment by ajayojha** [comment]
@Antoniossss & @ristomatti it's not coming in my local environment, which IDE you are using?

**Comment by Antoniossss** [comment]
IntelliJ IDEA 2020.1.4 (Ultimate Edition)
Build #IU-201.8743.12, built on July 21, 2020

**Comment by ristomatti** [comment]
@ajayojha Latest stable WebStorm. So it seems @Antoniossss and I have an IDE from the same vendor. JetBrain's IDE's do some type inference themselves but I wonder if this is a bug in their side or not. I assume you're using VSCode?

**Comment by ajayojha** [comment]
@ristomatti Yes, I am using VSCode and the message is not showing in the VSCode. I believe it's an IDE related issue.

**Comment by ajayojha** [comment]
@ristomatti & @Antoniossss I am closing this ticket as no action required from our end.
